libdnf.spec

```diff
@@ -191,6 +193,16 @@ Obsoletes:      platform-python-hawkey < %{version}-%{release}
 Python 3 bindings for the hawkey library.
 %endif
 
+%if %{with repoconfig_daemon}
+%package -n dnf-repoconfig-daemon
+Summary:        D-bus daemon for repository management
+BuildRequires:  pkgconfig(sdbus-c++)
+BuildRequires:  systemd-rpm-macros
+
+%description -n dnf-repoconfig-daemon
+D-bus daemon for listing, enabling and disabling repositories.
```

Comment:
Uh, is there going to be a bunch of daemons (one for repos, one for packages...)?

Reply:
No, it's not. This is just proof of concept and eventually there will be just one dnf daemon.